packages/core/addon/components/navi-visualizations/line-chart.js

```diff
@@ -230,8 +230,8 @@ export default Component.extend({
       options = merge({}, DEFAULT_OPTIONS, get(this, 'options')),
       curve = options.type.curve,
       area = options.type.area;
-    if (chartType == 'line') {
-      if (curve == 'line') {
+    if (chartType === 'line') {
+      if (curve === 'line') {
         return area ? 'area' : 'line';
       } else if (curve == 'spline' || curve == 'step') {
```

Comment:
missed a couple

Reply:
hmm need to look into why eslint isn't picking it up, rule might be turned off